DEVELOPMENT.md

```diff
@@ -34,6 +34,18 @@ We don't keep around a copy of an SDL schema file as they tend to get outdated
 quickly and it is way easier to explore the API via introspection. You can read
 more on that in the [GraphiQL Explorer](#graphiql-explorer) section below.
 
+If you need the schema to guide you writing new queries or mutations, you can
+get the latest version by querying the proxy via `apollo`:
+
+```sh
+# This installs apollo globally, we don't want to pull in all the extra
+# dependencies just for the off chance it is needed.
+yarn global add apollo
```

Comment:
Can't we declare it a dev dependency?

Reply:
Sure.

Reply:
GitHub's acting up, for some reason it doesn't show the latest commit in this PR: https://github.com/radicle-dev/radicle-upstream/commit/9405d61afab76068aa6ec1d699194168bcdd8ee6